data/free worlds middle.txt

```diff
@@ -1996,15 +1997,23 @@ mission "FW Defend New Tibet"
 			`	"That's insane," says Freya.`
 			`	"But not completely implausible," says JJ.`
 			label evidence
-			`	The monk says, "He brought a great deal of evidence. Plans for a nuclear device. Data from underground testing. A manufactured bomb casing. Samples of fissile material. And he named the planet where the development was done: Mutiny, in the Gienah system. We hid his ship in a cave. A day later, a Syndicate fleet appeared in orbit. They do not know where the defector landed, but they say they will destroy whatever settlement is harboring him unless he is returned to them immediately."`
+			`	The monk says, "He was unable to show us his evidence, but he made claims of having plans for a nuclear device. After arriving he hid his ship in a cave, and a Syndicate fleet appeared in orbit shortly afterward. They do not know where the defector landed, but they have begun searching settlement to settlement believing that one may be harboring him. They are currently here, wrecking the monastery in search of him."`
 			`	After a moment, JJ says, "If these allegations are true, if we can prove that the Free Worlds was innocent of any involvement in those attacks, the Republic might make peace with us, or at least agree to a ceasefire. I vote that we try to shelter this defector and hear his story."`
-			`	"And if it's a lie," says Freya, "and we refuse the Syndicate's demands, we lose our only ally. And many civilians will be killed, as well. I care about justice, but trading a definite ally for a slim possibility of peace: that is an unwise investment. I vote that we hand this man over to the Syndicate."`
-			`	The monk says, "We want to see justice done, and peace prevail. If you ask us to, we will flee to the mountains and take refuge until help arrives."`
+			`	"And if it's a lie," says Freya, "and we refuse the Syndicate's demands, we lose our only ally. We don't even know what the evidence is. And many civilians will be killed, as well. I care about justice, but trading a definite ally for a slim possibility of peace: that is an unwise investment. I vote that we hand this man over to the Syndicate."`
+			`	You look back at the video screen to see that the monk is gone, replaced by a man in a Syndicated Security uniform. "Greetings, Councilmembers."`
+			choice
+				`	"What is the meaning of this?"`
+				`	"Is the Syndicate really behind the bombings?"`
+			`	"I'm sorry for the confusion," the man responds, "but please understand. We're hunting a terrorist. This man claiming that we are the ones that caused the bombings is in fact the leader of the extremist group that caused them. He developed the nukes with the help of a faction of pirates looking to profit from the instability, and now that we've discovered that, he's attempting to run and frame us for his crimes against humanity. Allow us to find him, and we'll--"`
+			`	The feed cuts off. "I've lost the signal. It looks like it's been jammed by the Navy," the communications officer says.`
```

Comment:
A: This pushes the player to think the navy is somehow at wrong, or being framed.
B: How would you know who was jamming a signal.

Reply:
A: Is that an issue?
B: Idk, ask the communications officer.

Reply:
Think of the shock when during checkmate when the player gets cut off, while some in system signals might be disrupted clearly the idea that a signal might get interrupted on route is very unusual. 
I imagine it like the internet today, without cutting off everything everywhere it will almost always get a signal through. And even if it is just an educated guess saying it like that makes it sound obvious. If you think it is the navy then you could say "probably the navy" or "I think it's the navy", assuming that this guy is behaving vaguely professionally then he probably isn't going to speculate without facts, anyone can do that. Regardless, communication is something that seems to be taken for granted in this universe, at least in inhabited regions.

Reply:
The difference with when the Pug show up is that they can't connect to *anywhere* except the neighboring system. In this instance, it seems to be just this communication that's dropped.